README.md

```diff
@@ -107,12 +107,206 @@ See
+Requires Admin permission
+
+-   *alerts_enabled:* whether to enable or disable notifications.
+
+-   *notify_org_owner:* whether to notify organizations owners on projects reaching threshold.
+
+### budget_alerts_targets_update(targets)
+
+Update the current budget alerts settings with additional email targets per budget label
+Requires Admin permission
+
+-   *targets:* dictionary of budget labels and list of email addresses
+
+### active_billing_tags_list()
+
+Get a list of active billing tags
+Requires Admin permission
+
+### billing_tags_create(tags_list)
+
+Create a list of active billing tags
+Requires Admin permission
+
+-   *tags_list:* list of billing tags names to be created
+
+### active_billing_tag_by_name(name)
+
+Get detailed info on active or archived billing tag
+Requires Admin permission
+
+-   *name:* name of existing billing tag
+
+### billing_tag_archive(name)
+
+Archive an active billing tag
+Requires Admin permission
+
+-   *name:* name of existing billing tag to archive
+
+### billing_tag_settings()
+
+Get the current billing tag settings
+Requires Admin permission
+
+### billing_tag_settings_mode()
+
+Get the current billing tag settings mode
+Requires Admin permission
+
+### billing_tag_settings_mode_update(mode)
+
+Update the current billing tag settings mode
+Requires Admin permission
+
+-   *mode:* new mode to set the billing tag settings (see BillingTagSettingMode)
+
+### project_billing_tag(project_id)
+
+Get a billing tag assigned to a particular project by project id
+Requires Admin permission
+
+-   *project_id:* id of the project to find assigned billing tag
+
+###  project_billing_tag_update(billing_tag, project_id)
+
+Update project's billing tag with new billing tag.
+Requires Admin permission
+
+-   *billing_tag:* billing tag to assign to a project
+
+-   *project_id:* id of the project to assign a billing tag
+
+### project_billing_tag_reset(project_id)
+
+Remove a billing tag from a specified project
+Requires Admin permission
+
+-   *project_id:* id of the project to reset billing tag field
+
+### projects_by_billing_tag( billing_tag, offset, page_size, checkpoint_project_id, name_filter, sort_by, sort_order, missing_tag_only=False)
+
+Remove a billing tag from a specified project
+Requires Admin permission
+
+-   *billing_tag:* billing tag string to filter projects by
+
+-   *offset:* The index of the start of the page, where checkpointProjectId is index 0.
+If the offset is negative the project it points to will be the end of the page.
+-   *page_size:* The number of record to return per page.
+
+-   *checkpoint_project_id:* (Optional) project id that marks the start of where to look for the desired page from
+
+-   *name_filter:* matches projects by name substring
+
+-   *sort_by:* (Optional) field to sort the projects on
+
+-   *sort_order:* (Optional) Whether to sort in asc or desc order
+
+-   *missing_tag_only:* (Optional) determine whether to only return projects with missing tag
```

Comment:
Should we make a separate method to simply get all the projects without a billing tag?

Reply:
No, I don't think so, as the admin may still want to apply other filters.